core/src/main/java/com/scalar/db/transaction/consensuscommit/PrepareMutationComposer.java

```diff
@@ -51,68 +50,92 @@ public void add(Operation base, @Nullable TransactionResult result) {
   }
 
   private void add(Put base, @Nullable TransactionResult result) {
-    Put put =
-        new Put(base.getPartitionKey(), base.getClusteringKey().orElse(null))
-            .forNamespace(base.forNamespace().get())
-            .forTable(base.forTable().get())
-            .withConsistency(Consistency.LINEARIZABLE);
+    PutBuilder.Buildable putBuilder =
+        Put.newBuilder()
+            .namespace(base.forNamespace().get())
+            .table(base.forTable().get())
+            .partitionKey(base.getPartitionKey())
+            .consistency(Consistency.LINEARIZABLE);
+
+    if (base.getClusteringKey().isPresent()) {
+      putBuilder.clusteringKey(base.getClusteringKey().get());
+    }
```

Comment:
```suggestion
    base.getClusteringKey().ifPresent(putBuilder::clusteringKey);
```
It is detail but this can be simplified.